bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -268,6 +269,23 @@ public void run() {
                     logger.debug("Changed ThingType of Thing {} to {}. New ThingHandler is {}.",
                             thing.getUID().toString(), thing.getThingTypeUID(), thing.getHandler().toString());
                 }
+
+                private void waitForRunningHandlerRegistrations(ThingUID thingUID) {
+                    for (int i = 0; i < 10 * 10; i++) {
```

Comment:
Hi, why are you using 10 * 10 instead of 100. This stuff should be eliminated by the Java byte compiler, so I will not complain about it, but I don't understand the reason. Could you explain it to me?